### ISSUE TITLE
[BUG] Text area fields in forms doesn't receive prepopulated values

### ISSUE THREAD

**Issue by inseti** [mautic/mautic#1022]
Using version 2.1 when prepopulating field values in forms there is a bug:

If the field in the form is Textarea type it doesn't receive the value.

But changing the field form type from Textarea to Text, the value is received properly.

**Comment by inseti** [comment]
Sorry Alan, I've implemented the PR but the bug persists, the textarea field doesn't receive the "field value" when prepopulating (I'm not speaking about attribute value but field value)

I checked that the value is in the URL and field name is also ok.

If I set the form field as Text it works properly but the data is without line breaks and it's not possible to edit it properly

Thanks

**Comment by alanhartless** [comment]
I don't think I'm quite following what the issue is. Can you layout step by step what I need to do to experience the bug?

Is this when using the URL the form is embedded in to pre populate values? 

Thanks!

**Comment by inseti** [comment]
The aim is that a lead could confirm/modify his own lead data in a form with his values pre populated.

To reproduce the issue you must:

- Define at least one lead field which Data type is Textarea (i.e "description")
- Create a form matching the lead fields:
``` 
<div id="mauticform_form_email"  class="mauticform-row mauticform-email mauticform-required">
        <label id="mauticform_label_form_email" for="mauticform_input_form_email" class="mauticform-label">email</label>
         <input id="mauticform_input_form_email" name="mauticform[email]" value="" class="mauticform-input" type="email" />
 </div>
<div id="mauticform_form_description"  class="mauticform-row mauticform-text">
        <label id="mauticform_label_form_description" for="mauticform_input_form_description" class="mauticform-label">description</label>
       <textarea id="mauticform_input_form_description" name="mauticform[description]" value="" class="mauticform-textarea"></textarea>
 </div>
```
- Put that form in a landing page
- Send an email to a lead that include in the content the link to the landing page:
 ```{pagelink=1}&email={leadfield=email|true}&description={leadfield=description|true}```
- Click the link in the email received

Then you will see that, in the landing page with the form, the field "description" is not filled with its value.

If in the form, you change the field type for "description" and you choose "text" instead "textarea" and send the email again, the value is properly filled.

So the problem must be with textarea fields.

I hope this help!

**Comment by alanhartless** [comment]
Oh! Gotcha.  Ok, try the PR now. I found and I think fixed what you're referring to.  Along with adding support for checkboxes and radio groups.

**Comment by inseti** [comment]
+1 It works like a charm !!

**Comment by alanhartless** [comment]
Sorry @inseti, looks like we somehow managed to overlook the PR when we released and so it didn't make it into 1.2.1.  We'll get it into the next release.  You might have to re-patch after updating.

**Comment by igisho** [comment]
This is not working in 1.2.2 even on input text field. what is the fix for it?

**Comment by igisho** [comment]
how can i fix this issue? still presists in 1.2.2
